pkg/cmd/admin/groups/sync/rfc2307/ldapinterface.go

```diff
@@ -57,6 +59,9 @@ type LDAPInterface struct {
 	// cachedUsers holds the result of user queries for later reference, indexed on user UID
 	// e.g. this will map an LDAP user UID to the LDAP entry returned from the query made using it
 	cachedUsers map[string]*ldap.Entry
+
+	// Err is used to log errors found interfacing with LDAP
+	Err io.Writer
```

Comment:
while we're plumbing, I think I'd rather have an error handler here that would take the member lookup error (so it had all the context of the group uid, member uid, and error) and return whether to continue or fail